packages/integration-tests/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "devDependencies": {
     "@playwright/test": "^1.23.2",
-    "@types/node": "16",
+    "@types/node": "18.6.3",
```

Comment:
+1 I assume this should stay at something like `^16...`, also in `packages/server` it's still set to `"16"`